serving/backend/calculate_level.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import pandas as pd
+
+from training.creating_dataset import standardize_bestiary
+from training.save_load_model_to_file import load_model_from_file
+
+
+def calculate_level(monster_json: str, standardized: bool = False) -> str:
+    """
+    Calculates the monster's level based on its statistics
+    :param monster_json: JSON string with monster's statistics to load
```

Comment:
u nas cechy potworów zawsze są jako dict na serwerze (a nie plik) - nawet jak użytkownik wgrywa plik, to jest on od razu otwierany i wyciągane są z niego interesujące nas informacje (dla referencji jeszcze jako work in progress: [link](https://github.com/Paulina100/ML-for-TTRPG-game-design/blob/ba6d54db17efb08fa4b904485da0427b040b5bbc/serving/server.py#L32)). Dzieje się tak dlatego, że strona frontendowa będzie sobie pobierać te wypakowane properties i wrzucać je do formularza na cechy potwora, żeby użytkownik mógł sobie je edytować wedle uznania

Reply:
Hm, w przypadku formularza, w którym użytkownik ręcznie uzupełnia wartości properties, dane z reacta są przesyłane do fastapi jako słownik (json). Jak użytkownik wgrywa plik, to cały plik jest przesyłany i fastapi sobie go rozpakowuje i przegląda. Zrobiłam to głównie po to, żeby nie musieć szukać jak w jsie jest zrobione takie otwieranie plików, ale oczywiście mogę to jeszcze zmienić. Tylko nie jestem pewna, czy dobrze rozumiem, według Ciebie najlepiej by było (w obu przypadkach) w react-cie stworzyć plik json i jego przesyłać? Czy to nie przerost formy nad treścią, jak możemy przesłać po prostu słownik (nie wrzucony do pliku)?

Reply:
Tak, frontend przesyła słownik (oprócz przypadku, kiedy użytkownik przesyła plik, bo jego wygodniej mi było parsować w pythonie w fastapi)